src/components/Property/RepairsHistory/RepairsHistoryView.js

```diff
@@ -33,13 +41,28 @@ const RepairsHistoryView = ({ propertyReference, tabName }) => {
 
   useEffect(() => {
     setLoading(true)
+    setPageNumber(pageNumber + 1)
 
-    getRepairsHistoryView(propertyReference)
+    getRepairsHistoryView(propertyReference, pageNumber)
   }, [])
 
+  const handleShowMoreClick = (newPageNumber) => {
```

Comment:
Is there a better name for this perhaps? i.e. `handleLoadMoreWorkOrders` or even `loadMoreWorkOrders` ?
What do you reckon.